### ISSUE TITLE
Add support for JeOS

### ISSUE THREAD

**Issue by stdevel** [stdevel/ansible-uyuni#20]
In accordance with the [official documentation](https://documentation.suse.com/external-tree/en-us/suma/4.0/suse-manager/installation/install-vm.html) it would be good to implement the following "hack" to replace the base product with SUSE Manager:

```sh
# rpm -e --nodeps sles-release
# vi /etc/zypp/zypp.conf
...
solver.onlyRequires = false

ESC ZZ
# SUSEConnect -p SUSE-Manager-Server/4.0/x86_64 -r <code> -e <mail>
# SUSEConnect -p sle-module-basesystem/15.1/x86_64
# SUSEConnect -p sle-module-python2/15.1/x86_64
# SUSEConnect -p sle-module-server-applications/15.1/x86_64
# SUSEConnect -p sle-module-web-scripting/15.1/x86_64
# SUSEConnect -p sle-module-suse-manager-server/<productnumber>/x86_64
# ln -s /etc/products.d/SUSE-Manager-Server.prod /etc/products.d/baseproduct
# zypper in yast2-storage-ng
```

**Comment by stdevel** [comment]
Investigating in branch `feat-jeos-20`

**Comment by stdevel** [comment]
Fixed in commit cede07ed483990ae3e6a9464174081183d46e3a9.
